inst/Description.qml

```diff
@@ -16,18 +16,21 @@ Description
 	Analysis
 	{
 		title:	qsTr("Principal Component Analysis")
-		func:	"PrincipalComponentAnalysis"
+		func:	"principalComponentAnalysis"
+		qml:	"PrincipalComponentAnalysis.qml"
 	}
 
 	Analysis
 	{
 		title:	qsTr("Exploratory Factor Analysis")
-		func:	"ExploratoryFactorAnalysis"
+		func:	"exploratoryFactorAnalysis"
+		qml:	"ExploratoryFactorAnalysis.qml"
 	}
 
 	Analysis
 	{
 		title:	qsTr("Confirmatory Factor Analysis")
-		func:	"ConfirmatoryFactorAnalysis"
+		func:	"confirmatoryFactorAnalysis"
```

Comment:
ahh, wait, but the CFA R function remained `ConfirmatoryFactorAnalysis`, right?

Reply:
good catch